lib/ext/jiff-client-bignumber.js

```diff
@@ -221,7 +221,7 @@
       return jiff.helpers.BigNumber(v1).eq(v2);
     };
     jiff.share_helpers['floor/'] = function (v1, v2) {
-      return v1.div(v2).floor();
+      return v1.div(v2).decimalPlaces(0, 3);
```

Comment:
return v1.div(v2).decimalPlaces(0, 3); the second arg represent to which direction to round and 3 corresponds to to round floor.

In theory, you should be able to jiff.helpers.BigNumber.ROUND_FLOOR, but it turned out to be failing to fetch 3 in jiff_lagrange in the jiff-client-bignumber.js for some reason.